### PR TITLE
Version bumped 1.3

### PR DESCRIPTION
**Pull request by AlexandrePTJ**

### CODE REVIEW

**Comment by SSE4** [comment]
Windows fails, need to apply this patch: https://github.com/fnc12/sqlite_orm/pull/159/files

**Comment by SSE4** [comment]
@AlexandrePTJ do you have an update on this one?
please also link discussion here, if possible

**Comment by AlexandrePTJ** [comment]
Yes, repository's tags have been cleaned up. Then now it works as expected.

File: conanfile.py [hunk 1]
```diff
@@ -3,30 +3,39 @@
...
-    version = "1.2"
+    version = "1.3"
     description = "SQLite ORM light header only library for modern C++."
     url = "https://github.com/bincrafters/conan-sqlite_orm"
     homepage = "https://github.com/fnc12/sqlite_orm"
+    topics = ("conan", "sqlite", "sql", "database", "orm", "sqlite_orm")
     author = "AlexandrePTJ <alpetitjean@gmail.com>"
     license = "BSD 3-Clause"
     exports = ["LICENSE.md"]
+    exports_sources = ["patches/*.diff"]
```
Comment (uilianries) [inline conanfile.py:19]: where are these patches?

**Review by uilianries — CHANGES_REQUESTED** [review]
Some patches are listed in the recipe, but there are no files in the repository.

**Comment by uilianries** [comment]
@AlexandrePTJ Could you explain where are those patch files?

File: conanfile.py (hunk 1 above)
Comment (AlexandrePTJ) [reply conanfile.py:19]: I missed this one. No more patch are necessary, this line must be delete.

**Pull request closed without merging**